B/Blake2/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Blake2"
+version = v"0.98.1"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/BLAKE2/libb2.git" =>
+    "f4eedd3cde347127dfc9b85cb4ea7fecec75df44",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libb2/
+./autogen.sh 
+./configure --prefix=${prefix} --host=${target}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libb2", :Blake2Lib)
```

Comment:
What about calling the variable `libblake2` instead?

Reply:
sure. I don't mind changing the variable name
since everything in this pull request was created by run_wizard() of BinaryBuilder, how can edit this file without (change variable name) without rerunning run_wizard()?

Reply:
The pull request was opened from your fork, that maybe the wizard automatically created for you.  You can clone your fork locally and push the changes to the branch `wizard/Blake2-v0.98.1_6eab5a9b`.  Anyway, I can do that for you if you're not comfortable with git.